[python3:x64-linux] build failure

**Host Environment**
- OS: `openSUSE Tumbleweed`
- Compiler: `gcc (SUSE Linux) 10.2.1 20200805 [revision dda1e9d08434def88ed86557d08b23251332c5aa]`

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install python3`

**Failure logs** 
```log
-- Installing x64-linux-rel share files...
-- Installing x64-linux-rel Python library files...
-- Installing x64-linux-rel done
-- Configuring x64-linux-dbg
-- Building x64-linux-dbg
-- Installing x64-linux-dbg Python library files...
-- Installing x64-linux-dbg done
-- Installing: /home/valery/Cache/vcpkg/packages/python3_x64-linux/share/python3/copyright
-- Performing post-build validation
There should be no empty directories in /home/valery/Cache/vcpkg/packages/python3_x64-linux
The following empty directories were found:

    /home/valery/Cache/vcpkg/packages/python3_x64-linux/lib
    /home/valery/Cache/vcpkg/packages/python3_x64-linux/debug/lib

If a directory should be populated but is not, this might indicate an error in the portfile.
If the directories are not needed and their creation cannot be disabled, use something like this in the portfile to remove them:

    file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/a/dir" "${CURRENT_PACKAGES_DIR}/some/other/dir")


Found 1 error(s). Please correct the portfile:
    /home/valery/Cache/vcpkg/ports/python3/portfile.cmake
-- Performing post-build validation done
Error: Building package python3:x64-linux failed with: POST_BUILD_CHECKS_FAILED
Please ensure you're using the latest portfiles with `./vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: python3:x64-linux
  Vcpkg version: 2020.06.15-unknownhash

Additionally, attach any relevant sections from the log files above.
```

Ok, sure. Here is the issue from `boost-python`: 
```
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:141 (message):
    Command failed: /home/valery/Cache/vcpkg/downloads/tools/cmake-3.17.2-linux/cmake-3.17.2-Linux-x86_64/bin/cmake --build . --config Debug --target install -- -v
    Working Directory: /home/valery/Cache/vcpkg/buildtrees/boost-python/x64-linux-dbg
    See logs for more information:
      /home/valery/Cache/vcpkg/buildtrees/boost-python/install-x64-linux-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_cmake.cmake:91 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_cmake.cmake:24 (vcpkg_build_cmake)
  installed/x64-linux/share/boost-build/boost-modular-build.cmake:114 (vcpkg_install_cmake)
  installed/x64-linux/share/boost-build/boost-modular-build.cmake:119 (unix_build)
  ports/boost-python/portfile.cmake:12 (boost_modular_build)
  scripts/ports.cmake:79 (include)


Error: Building package boost-python:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `./vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: boost-python:x64-linux
  Vcpkg version: 2020.06.15-unknownhash
```
I can make another issue for this

--- a/ports/python3/CONTROL
+++ b/ports/python3/CONTROL
@@ -1,6 +1,6 @@
 Source: python3
 Version: 3.8.3
-Port-Version: 1
+Port-Version: 2
 Homepage: https://github.com/python/cpython
 Description: The Python programming language as an embeddable library
 Build-Depends: libffi, openssl, zlib (!uwp&!windows)

--- a/ports/python3/portfile.cmake
+++ b/ports/python3/portfile.cmake
@@ -161,17 +161,16 @@ elseif (VCPKG_TARGET_IS_LINUX OR VCPKG_TARGET_IS_OSX)
 		)
 
 		message(STATUS "Installing ${TARGET_TRIPLET}-rel lib files...")
-		file(GLOB LIBS
+		file(GLOB PY_LIBS
 			${OUT_PATH_RELEASE}/lib/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/*)
-		file(INSTALL ${LIBS} DESTINATION ${CURRENT_PACKAGES_DIR}/share/python${PYTHON_VERSION_MAJOR}/Lib
+		file(INSTALL ${PY_LIBS} DESTINATION ${CURRENT_PACKAGES_DIR}/share/python${PYTHON_VERSION_MAJOR}/Lib
 			PATTERN "*.pyc" EXCLUDE
 			PATTERN "*__pycache__*" EXCLUDE
 		)
 
 		message(STATUS "Installing ${TARGET_TRIPLET}-rel share files...")
-		file(GLOB LIBS
-			${OUT_PATH_RELEASE}/lib/pkgconfig/*)
-		file(INSTALL ${LIBS} DESTINATION ${CURRENT_PACKAGES_DIR}/share/python${PYTHON_VERSION_MAJOR}
+		file(GLOB PKGCFG ${OUT_PATH_RELEASE}/lib/pkgconfig/*)
+		file(INSTALL ${PKGCFG} DESTINATION ${CURRENT_PACKAGES_DIR}/share/python${PYTHON_VERSION_MAJOR}
 			PATTERN "*.pyc" EXCLUDE
 			PATTERN "*__pycache__*" EXCLUDE
 		)
@@ -179,6 +178,9 @@ elseif (VCPKG_TARGET_IS_LINUX OR VCPKG_TARGET_IS_OSX)
 		message(STATUS "Installing ${TARGET_TRIPLET}-rel Python library files...")
 		file(GLOB LIBS
 			${OUT_PATH_RELEASE}/lib/libpython${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}.*)
+        if (NOT LIBS)
+            file(GLOB LIBS ${OUT_PATH_RELEASE}/lib64/*)
+        endif()
 		file(INSTALL ${LIBS} DESTINATION ${CURRENT_PACKAGES_DIR}/lib
 			PATTERN "*.pyc" EXCLUDE
 			PATTERN "*__pycache__*" EXCLUDE
@@ -233,6 +235,10 @@ elseif (VCPKG_TARGET_IS_LINUX OR VCPKG_TARGET_IS_OSX)
 		message(STATUS "Installing ${TARGET_TRIPLET}-dbg Python library files...")
 		file(GLOB LIBS
 			${OUT_PATH_DEBUG}/lib/libpython${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}d.*)
+        if (NOT LIBS)
+            file(GLOB LIBS
+                ${OUT_PATH_DEBUG}/lib64/*)
+        endif()
 		file(INSTALL ${LIBS} DESTINATION ${CURRENT_PACKAGES_DIR}/debug/lib
 			PATTERN "*.pyc" EXCLUDE
 			PATTERN "*__pycache__*" EXCLUDE

